src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/storage/LeaseAcquirer.java

```diff
@@ -32,16 +32,36 @@ public LeaseAcquirer(
 
     /**
      * Main wrapper for blobs to be leased by {@link BlobLeaseClient}.
+     *
      * @param blobClient Represents blob
      * @param onLeaseSuccess Consumer which takes in {@code leaseId} acquired with {@link BlobLeaseClient}
      * @param onFailure Extra step to execute in case an error occurred
-     * @param releaseLease Flag weather to release the lease or not
+     * @param releaseLease Flag whether to release the lease or not
      */
     public void ifAcquiredOrElse(
         BlobClient blobClient,
         Consumer<String> onLeaseSuccess,
         Consumer<BlobErrorCode> onFailure,
         boolean releaseLease
+    ) {
+        ifAcquiredOrElse(blobClient, onLeaseSuccess, onFailure, releaseLease, true);
```

Comment:
will it ever be false?

Reply:
yes, OCR validation retry properties should be kept after releasing lease. But this PR has been updated - only lease related metadata will be clered.